t/debugger.lisp

```diff
@@ -57,7 +57,10 @@
 
 (deftest debugger.condition.1
   (eqt (first (portable-condition-system::run-debugger-command
-               :condition nil *debugger.condition*))
+               :condition
+               ;; Dummy stream that supports both input and output
+               (make-two-way-stream (make-concatenated-stream) (make-broadcast-stream))
```

Comment:
Does it mean that passing `nil` to `run-debugger-command` is now unsupported?

If yes, then this stream needs to be created internally, and changes to this test case should be reverted - we should support having `nil` as a valid argument when no streams are required from the outside, because the user does not always have a meaningful stream to pass.

Reply:
Oh okay, can do; it will cause a condition to be signalled in the current state if nil is passed